app.rb

```diff
@@ -27,14 +27,34 @@ def search_enabled?
   GC::Profiler.enable
 end
 
+def get_logger(progname, threshold=nil)
+  logger = Logger.new(STDERR)
+  logger.progname = progname
+  logger.level = threshold || Logger::INFO
+  logger
+end
+
 application_yaml = ERB.new(File.read("config/application.yml")).result()
 CommentService.config = YAML.load(application_yaml).with_indifferent_access
 
+# Raise sinatra-param exceptions so that we can process, and respond to, them appropriately
+set :raise_sinatra_param_exceptions, true
+
+# Setup Mongo
 Mongoid.load!("config/mongoid.yml", environment)
 Mongoid.logger.level = Logger::INFO
 Mongo::Logger.logger.level = ENV["ENABLE_MONGO_DEBUGGING"] ? Logger::DEBUG : Logger::INFO
 
-# set up i18n
+# Setup Elasticsearch
+# NOTE (CCB): If you want to see all data sent to Elasticsearch (e.g. for debugging purposes), set the tracer argument
+# to the value of a logger.
+# Example: Elascisearch.Client.new(tracer: get_logger('elasticsearch.tracer'))
+Elasticsearch::Model.client = Elasticsearch::Client.new(
+    host: CommentService.config[:elasticsearch_server],
+    logger: get_logger('elasticsearch', Logger::WARN)
+)
```

Comment:
This is a repeat of: https://github.com/edx/cs_comments_service/pull/225/files#diff-cc95738088603531796e0d0f246a5d77L54
However, since this configures logging, we probably want to remove the older line.